src/Runtimes/Runtimes.php

```diff
@@ -75,16 +80,19 @@ public function __construct($version = '')
         $java->addVersion('11.0', 'openjdk/11-jdk-slim', 'openruntimes/java:' . $this->version . '-11.0', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
         $java->addVersion('17.0', 'openjdk/17-jdk-slim', 'openruntimes/java:' . $this->version . '-17.0', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
         $java->addVersion('18.0', 'openjdk/18-jdk-slim', 'openruntimes/java:' . $this->version . '-18.0', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
+        $java->addVersion('21.0', 'openjdk/21-jdk-slim', 'openruntimes/java:' . $this->version . '-21.0', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
         $this->runtimes['java'] = $java;
 
         $swift = new Runtime('swift', 'Swift', 'src/function/Runtime serve --env production --hostname 0.0.0.0 --port 3000');
         $swift->addVersion('5.5', 'swiftarm/swift:5.5.3-ubuntu-jammy', 'openruntimes/swift:' . $this->version . '-5.5', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
         $swift->addVersion('5.8', 'swiftarm/swift:5.8-ubuntu-jammy', 'openruntimes/swift:' . $this->version . '-5.8', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
+        $swift->addVersion('5.9', 'swift:5.9-jammy', 'openruntimes/swift:' . $this->version . '-5.9', [System::X86, System::ARM64,  System::ARMV7, System::ARMV8]);
```

Comment:
why not `ubuntu-jammy` ?

Reply:
I think Luke just copied what he saw in Open RUntimes. swift image as base image should be fine. We do that for most if not all runtimes - we get image that has runtime pre-installed.

Reply:
We used `swift:5.9-jammy` as the base image here: https://github.com/open-runtimes/open-runtimes/blob/main/runtimes/swift-5.9/Dockerfile

It should be the same tag as `ubuntu-jammy` but wanted to keep consistent with base image